src/app/src/main/resources/metadata/charts/cumulative_percentage_of_population_vaccinated/input.schema.json

```diff
@@ -1,46 +1,39 @@
 {
   "type": "object",
   "properties": {
-    "cumulative_percent_pop_vacced": {
+    "cumulativePercentPopulationVaccinated": {
       "type": "array",
       "items": {
         "type": "object",
         "properties": {
           "month": {
             "type": "string"
           },
-          "healthcare_workers": {
+          "healthcareWorkers": {
             "type": "number"
           },
-          "over_65_years": {
+          "over65": {
             "type": "number"
           },
-          "working_age": {
+          "workingAge": {
             "type": "number"
           }
         },
         "required": [
           "month",
-          "healthcare_workers",
-          "over_65_years",
-          "working_age"
+          "healthcareWorkers",
+          "over65",
+          "workingAge"
         ]
       }
     },
-    "params": {
-      "type": "object",
-      "properties": {
-        "population": {
-          "type": "number"
-        }
-      },
-      "required": [
-        "population"
-      ]
+    "population": {
+      "type": "number"
     }
   },
   "required": [
-    "cumulative_percent_pop_vacced",
-    "params"
+    "cumulativePercentPopulationVaccinated",
+    "params",
```

Comment:
Yes, it can come out here, because population is not coming directly from params, it's a separate value from params (though it will depend on the region param). It's still used in other charts which use param values (provided to the hart component in 'layoutData'). I think there is value in distinguishing these, it makes it more obvious where data is coming from, and potentially avoids name clashes (though they ought to be avoidable by sensible naming anyway!)